src/surveys/components/SelectResponse.tsx

```diff
@@ -0,0 +1,85 @@
+import { VNode, h } from 'preact';
+import { useState } from 'preact/hooks';
```

Comment:
@afolasope Did you forget to delete this file?

Reply:
Yes, but I have deleted it now